programs/drift_vaults/src/instructions/request_withdraw.rs

```diff
@@ -28,7 +28,13 @@ pub fn request_withdraw<'info>(
 
     let n_shares: u128 = vault_amount_to_if_shares(amount, vault.total_shares, net_usd_value)?;
 
-    vault_depositor.request_withdraw(n_shares, net_usd_value, vault, clock.unix_timestamp)?;
+    vault_depositor.request_withdraw(
+        n_shares,
+        WithdrawUnit::Shares,
```

Comment:
was thinking we could let people pass in the withdraw using to ix

Reply:
yea this was just step 1